<launch>
   <!-- <node pkg="hrl_phri_2011" type="ellipsoid_visualizer" name="ellipsoid_visualizer"
          args="$(find hrl_phri_2011)/data/sub1_head_stitched.bag $(find hrl_phri_2011)/data/sub1_ellipsoid_registration.bag" />
   -->
    <node pkg="hrl_phri_2011" type="interactive_ellipse" name="interactive_ellipse"
          args="base_link ellipse_frame 8 $(find hrl_phri_2011)/data/cur_ellipsoid_registration.bag $(find hrl_phri_2011)/data/cur_ellipsoid_registration.bag 1" />
    <node pkg="hrl_rfh_fall_2011" type="head_tool_pose_server.py" name="head_tool_pose_server" />
    <node pkg="hrl_rfh_fall_2011" type="head_markers.py" name="head_markers" output="screen" />
    <node pkg="hrl_rfh_fall_2011" type="pc_throttled_switch" name="pc_throttled_switch" 
          output="screen" args="/kinect_head/rgb/points 8" />
</launch>
